packages/backend/src/db/migrations/20240702070156_add_composite_index_on_executions.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.table('executions', (table) => {
+    table.index(['flow_id', 'test_run', 'created_at'])
```

Comment:
[80% sure](https://github.com/knex/knex/issues/3913) we need `raw` if we want to build index concurrently. Our executions table is large-ish now; we should concurrently create or run this migration at 12 am :/

Reply:
actually i also rmb we had to add index to `executions` table last tine (for filtering execution status on UI) and it was damn fast right. so maybe doesn't matter

Reply:
hmm yea maybe we can do it in the evening in case